--- tilem-2.0/gui/macro.c.debug	2011-11-27 05:47:07.000000000 +0900
+++ tilem-2.0/gui/macro.c	2024-10-13 16:38:22.782439121 +0900
@@ -86,7 +86,7 @@ void tilem_macro_add_action(TilemMacro*
 	macro->actions = tilem_macro_actions_new(macro, n + 1);
 
 	/* Then we need to save the action */	
-	macro->actions[n] =  g_new(char, strlen(value)); /* FIXME : gcc says : "assignment from incompatible pointer type" ??? */
+	macro->actions[n] =  g_new(TilemMacroAtom, 1);
 	macro->actions[n]->value = g_strdup(value);
 	macro->actions[n]->type = type;
 	macro->n++;
